### PR TITLE
Feature/ewl 4215 migrate footer

### PR DESCRIPTION
**Pull request by chelsiejohnston**
## Ticket(s)

**Jira Ticket**

- [EWL-4215: SG2 | Migrate the Footer organism](https://issues.ama-assn.org/browse/EWL-4215)

## Description
Adds Twig template files, styling, and a .json data structure for the Footer organism pattern.

## To Test
* [ ] `gulp serve`
* [ ] Organisms > Footer (http://localhost:3000/?p=organisms-footer)
  * [ ] Observe you see a light gray footer
  * [ ] Observe you see an `h2` that read "The AMA promotes the art and science of medicine and the betterment of public health"
  * [ ] Observe you see the social follow icons and they are linked approrpriately
  * [ ] Observe you see a horizontal rule
  * [ ] Observe you see the footer menu in two columns
  * [ ] Observe you see the copyright information in small light gray text
  * [ ] Observe you see the utility menu links in small dark gray text
* [ ] Pages > Topic (http://localhost:3000/?p=pages-topic)
  * [ ] Observe you _do not_ see a placeholder for the footer
  * [ ] Observe you see the footer organism

## Visual Regressions
Compare against SG1 -> https://americanmedicalassociation.github.io/AMA-style-guide/?p=organisms-footer

_This update includes the new typography, which will effect typography and spacing throughout the element_

## Relevant Screenshots/GIFs
screen shot 2018-01-31 at 2 28 53 pm

## Remaining Tasks
- [x] Add a .md file
- [x] Add the organism to the Topic page pattern
- [ ] Review in D8 - https://github.com/AmericanMedicalAssociation/ama-d8/pull/251

## Additional Notes
I did not update the footer on the template pages. Since this is not the official footer for the AMAOne project, I left those as placeholders (and also left the placeholder on the news article page). 

**This is the SG2 counterpart to https://github.com/AmericanMedicalAssociation/ama-d8/pull/251**.

### CODE REVIEW

**Comment by designerneil** [comment]
I noticed that the padding and margins don't match up with the footer in SG1. Can we get them to match a bit more closely?

## SG1 Footer
screen shot 2018-02-01 at 10 45 41 am

## Current footer
screen shot 2018-02-01 at 10 45 35 am

File: styleguide/source/_patterns/03-organisms/footer.twig [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {% endblock %}
+    <div class="ama__rule ama__rule--horizontal"></div>
+
+    {% block menus %}
+      <div class="ama__footer__menu-secondary-user">
+        {% for column in menus.secondaryUser %}
+          <ul>
+            {% for link in column %}
+              <li>
+                <a href="{{ link.url }}">{{ link.text }}</a>
```
Comment (designerneil) [inline styleguide/source/_patterns/03-organisms/footer.twig]: These need to be bold and have more padding around them.

File: styleguide/source/_patterns/05-pages/topic.twig [hunk 2]
```diff
@@ -10,4 +10,8 @@
 
 {% block sidebarSecondary %}
   {% include '@base/placeholder.twig' with {'placeholder':{'text':'Page sidebar secondary content'}} %}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block footer %}
+  {% include '@organisms/footer.twig' %}
+{% endblock %}
```
Comment (designerneil) [inline styleguide/source/_patterns/05-pages/topic.twig:18]: need empty line

Comment (labbydev) [reply styleguide/source/_patterns/05-pages/topic.twig:18]: This has an empty line

**Comment by labbydev** [comment]
@designerneil thanks for flagging some of the issues at larger breakpoints. I didn't realize there wasn't a container in the footer. This should fix some things. There may still be small spacing issues but I'd like to resolve some of the smaller issues when we have UX review. Please review and LMK what you think.

**Comment by designerneil** [comment]
@labbydev much better. The only thing left is to a class of ama__link--black to all the links to remove the underline. Links be default have underlines in them if no class is applied to them.
screen shot 2018-02-01 at 3 21 40 pm

**Comment by labbydev** [comment]
Thanks @designerneil this should be fixed now.

**Pull request merged**